Frontend-Projects/KeyboardHero/style.css

```diff
@@ -0,0 +1,208 @@
+body {
+    background-color: black;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-weight: 500;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+  }
+  
+  .title {
+    color: mintcream;
+    text-transform: uppercase;
+    margin-top: 3em;
+    margin-bottom: 3em;
+    font-size: 1em;
+    letter-spacing: 0.3em;
+  }
+  
+  .keyboard {
+    display: flex;
+    flex-direction: column;
+    border: 2px solid white;
+    border-radius: 0.625rem;
+    background-color: rgb(201, 156, 105);
+  }
+  
+  .row {
+    list-style: none;
+    display: flex;
+    padding: 0 20px;
+  }
+  
+  li {
+    height: 3em;
+    width: 3em;
+    color: rgba(0,0,0,0.7);
+    border-radius: 0.4em;
+    line-height: 3em;
+    letter-spacing: 1px;
+    margin: 0.4em;
+    transition: 0.3s;
+    text-align: center;
+    font-size: 1em;
+    border: 1px solid black !important
+  }
+  
+  #tab {
+    width: 5em;
+  }
+  
+  #caps {
+    width: 6em;
+  }
+  
+  #left-shift {
+    width: 8em;
+  }
+  
+  #enter {
+    width: 6em;
+  }
+  
+  #right-shift {
+    width: 8em;
+  }
+  
+  #back {
+    width: 5em;
+  }
+  
+  .pinky { 
+    background-color: crimson;
+    border: 2px solid crimson;
+  }
+  .pinky.selected {
+    color: crimson;
+  }
+  
+  .ring { 
+    background-color: coral;
+    border: 2px solid coral;
+  }
+  .ring.selected {
+    color: coral;
+  }
+  
+  .middle { 
+    background-color: darkorange;
+    border: 2px solid darkorange;
+  }
+  .middle.selected {
+    color: darkorange;
+  }
+  
+  .pointer1st { 
+    background-color: gold;
+    border: 2px solid gold;
+  }
+  .pointer1st.selected {
+    color: gold;
+  }
+  
+  .pointer2nd { 
+    background-color: khaki;
+    border: 2px solid khaki;
+  }
+  .pointer2nd.selected {
+    color: khaki;
+  }
+  
+  .fill-out-key {
+    background-color: slategrey;
+    border: 2px solid slategrey;
+  }
+  
+  .selected {
+    background-color: transparent;
+        -webkit-animation: vibrate-1 0.3s linear infinite both;
+              animation: vibrate-1 0.3s linear infinite both;
+  }
+  
+  /* ----------------------------------------------
+   * Generated by Animista
```

Comment:
Copied project

Reply:
So we can't contribute others open source projects?